torch/csrc/jit/script/compiler.cpp

```diff
@@ -563,6 +564,55 @@ struct to_ir {
     ConstantPooling(to_clean);
   }
 
+void runInline(Block* block) {
+
+  Node* cur = block->nodes().front();
+  Node* end = block->return_node();
+
+  while (cur != end)
+  {
+    auto next = cur->next();
+    if (cur->kind() == prim::CallModuleFunction)
+    {
+      AT_ASSERT(cur->inputs().at(0)->node()->kind() == prim::Constant);
+      auto function_constant = cur->inputs().at(0)->node();
+      const auto& fqfn = function_constant->s(attr::function_name);
+      auto mv = resolveFullyQualifiedFunctionName(fqfn, fakeRange());
+      auto& method = mv->getMethod();
+
+      auto old_output = cur->outputs();
+      //slice function ptr value
+      auto inputs = cur->inputs().slice(1);
+      WithInsertPoint guard(next);
+      auto new_output = inlineCallTo(*cur->owningGraph(), *method.graph().get(), inputs).at(0);
+      if (old_output.at(0)->hasUniqueName())
+      {
+        auto name = old_output.at(0)->uniqueName();
+        new_output->setUniqueName(name);
+      }
+
+      old_output.at(0)->replaceAllUsesWith(new_output);
+      next = cur->next();
+      cur->destroy();
```

Comment:
DCE won't touch this, since it's marked as having side effects, so we have to remove it manually.